package.json

```diff
@@ -101,7 +101,7 @@
     "bpk-component-icon": "5.0.22-css.0",
     "bpk-component-image": "3.0.22-css.0",
     "bpk-component-infinite-scroll": "3.0.23-css.0",
-    "bpk-component-input": "5.0.22-css.0",
+    "bpk-component-input": "^5.0.24-css.0",
```

Comment:
Can you remove the caret from this? It causes npm to resolve to the non css version when this is present at the beginning due to how it views the latest and pre-release versions